exercise6.rb

```diff
@@ -0,0 +1,36 @@
+require 'selenium-webdriver'
+
+class Exercise
+
+    def setup
+        Selenium::WebDriver::Firefox::Service.driver_path = "/home/vibhav/Desktop/INFUSE/driver/geckodriver"
+        @driver = Selenium::WebDriver.for :ff
+        @url = "https://letcode.in/forms"
+    end
+
+    def action
+        begin
+            @driver.get(@url)
+           
+            select = @driver.find_element(:tag_name, "select")
```

Comment:
Try doing this with select class, link sent in teams group

Reply:
Needed clarification

Reply:
Implemented using select class